v2/refs/marshal.go

```diff
@@ -123,6 +133,23 @@ func (o *ObjectID) StableSize() int {
 	return proto.BytesSize(objectIDValField, o.val)
 }
 
+// ObjectIDNestedListSize returns byte length of nested
+// repeated ObjectID field with fNum number.
+func ObjectIDNestedListMarshal(fNum int64, buf []byte, ids []*ObjectID) (off int, err error) {
```

Comment:
Can we reuse this function in `searchResponseBody.StableMarshal()`? It also marshals list of object IDs.

Reply:
Yes we can